### PR TITLE
Enable 32bit tests

### PR DESCRIPTION
**Pull request by xinaxu**
Enable 32bit tests as stream-commp 32bit issue is now fixed.
However, 32bit tests does not work on windows so let's just add another CI for 32-bit testing only on ubuntu

### CODE REVIEW

**Review by masih — COMMENTED** [review]
Would it make sense to instead get the [currently disabled](https://github.com/data-preservation-programs/singularity/blob/main/.github/workflows/go-test-config.json#L2) 32-bit tests in the unified CI going?

I imagine we want to entirely retire the custom CI setup at some point.

**Comment by xinaxu** [comment]
@masih 
I tried. The issue is that there is no working sqlite library that works in windows + 386.
So with unified CI, we have to either drop 386 or windows.

Then I think it's better to keep unified CI for windows and add a linux+386 test as a separate check

**Comment by masih** [comment]
> there is no working sqlite library that works in windows

To work around this we could:
* either drop test runs on windows as you suggested, or
* run the tests on Windows with a different DB backing, considering this repo uses ORM.

?

File: .github/workflows/go-test-config.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "skip32bit": true
+  "skip32bit": false
```
Comment (masih) [inline .github/workflows/go-test-config.json:2]: We can remove this file now.

**Review by masih — APPROVED** [review]

File: .github/workflows/go-test-config.json (hunk 1 above)
Comment (xinaxu) [reply .github/workflows/go-test-config.json:2]: I'll remove the file

**Pull request merged**